### PR TITLE
Scroll to top of Collections Artworks Grid after Applying Filters

### PR DESCRIPTION
**Pull request by ashleyjelks**
**Standard Collection**
Kapture 2020-05-11 at 14 54 24

**Department Collection**
Kapture 2020-05-11 at 14 55 46

https://artsyproduct.atlassian.net/browse/FX-1921

#trivial

### CODE REVIEW

**Comment by sweir27** [comment]
The implementation looks okay, but it feels a little jarring in the gifs! I wonder if we can make it slower? Nicole might have some animation specs/ideas. Maybe we can ease it?

**Comment by ashleyjelks** [comment]
> The implementation looks okay, but it feels a little jarring in the gifs! I wonder if we can make it slower? Nicole might have some animation specs/ideas. Maybe we can ease it?

Actually just posted a video in Slack of what it looks like with and without the animation just as you noted this!

File: src/lib/Scenes/Collection/Collection.tsx [hunk 1]
```diff
@@ -116,9 +134,14 @@ export class Collection extends Component<CollectionProps, CollectionState> {
                             <CollectionHubsRails linkedCollections={linkedCollections} {...this.props} />
                           ) : null
                         case "collectionArtworks":
+                          const scrollIndex = isDepartment ? 2 : 1
```
Comment (sweir27) [inline src/lib/Scenes/Collection/Collection.tsx:137]: If we're going to `- 1` above, can we just say `isDepartment ? 1 : 0` here? Nots sure if there's another use for the `scrollIndex` that I'm missing!

**Comment by sweir27** [comment]
Nice!! Looks awesome 😍!

**Pull request merged**